<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">toolbar{
background: rgb(90, 90, 90);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPSD"/>
     <addaction name="actionPNG"/>
     <addaction name="actionJPEG"/>
    </widget>
    <addaction name="actionCreate_new"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuLayer">
    <property name="title">
     <string>Layer</string>
    </property>
    <addaction name="actionClear_Layer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLayer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCreate_new">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionPSD">
   <property name="text">
    <string>.PSD</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>.PNG</string>
   </property>
  </action>
  <action name="actionJPEG">
   <property name="text">
    <string>.JPEG</string>
   </property>
  </action>
  <action name="actionClear_Layer">
   <property name="text">
    <string>Clear Layer</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
